### ISSUE TITLE
link Database Structure tab to selection

### ISSUE THREAD

**Issue by JackArcher2** [sqlitebrowser/sqlitebrowser#2483]
Have the 'Database Structure' tab be linked directly to what is selected when going into the 'Browse Data' tab.
For example, in the database structure tab if I have table1,table2, etc and I **select** table2, then I click the 'Browse Data' tab, I don't want to have to redundantly re-select under "table:" dropdown table2 again.
--------------------------

Additional info
---------------
Please answer these questions before submitting your feature request.

**Comment by chrisjlocke** [comment]
What version of DB4S are you using?  If you download the latest version (3.12.1) you'll find this works - as well as floaty docky windows.  

![image](https://user-images.githubusercontent.com/7872218/99777639-853d8180-2b0a-11eb-81d0-8ba1fab5a898.png)

Go here to download the latest version.  You didn't even mention what operating system you were using...
https://sqlitebrowser.org/dl/

**Comment by JackArcher2** [comment]
Running 3.12.1 x64 no installer version.

All this is minor and just a matter of usability. I understand that there are other ways to get the same results.

Let me rephrase it.
These are the steps to show how it isn't working.
1. click database structure tab
2. click and highlight any table **not** the first table
3. click browse data tab --> defaults to the first table

This works as what I had envisioned for the above:
1. click database structure tab
2. click and highlight any table **not** the first table
3. right click the table name and choose "Browse Table"
4. opens the browse data tab and goes to the correct table

**Comment by chrisjlocke** [comment]
Can you click 'Help' and 'About' and just check what version you're using.
The 'feature' you are explaining should already be in the 3.12.1 release.

The floaty docky windows isn't - just noticed that's in the nightlies.  But right-clicking a table in the 'Database structure' tab opens the right table in the 'Browse Data' tab.

**Comment by JackArcher2** [comment]
![Capture](https://user-images.githubusercontent.com/48897538/99828659-3a7f3200-2b29-11eb-8366-7338f928df51.JPG)

**Comment by chrisjlocke** [comment]
Any chance of copying over the create statements from the tables?  (right-click a table, select 'Copy create statement').  If the data isn't top-secret, you can email the database over to me (db4s [at] chrisjlocke [dot] co [dot] uk) or just the schema.  Just wondering if something wonky in the table names if breaking the selection routine.  
Are the table names 'generic' or do they contain weird characters?

**Comment by JackArcher2** [comment]
sent.

yes there are some entities in the data such as `&#x28;`

**Comment by chrisjlocke** [comment]
Received - thanks.
Still can't replicate the problem though.

https://streamable.com/fa0086

**Comment by JackArcher2** [comment]
Ok I see the problem in the video. You are doing the steps to recreate what works.

Here is the picture of how I am clicking


  1.  data structure tab
  2.  select second table - children
  3.  click  the browse data tab (not the right-click browse table option)

(attached)


________________________________

**Comment by chrisjlocke** [comment]
Aah yes - you have to do an 'action' - if you just change tabs, that could be annoying if it always flipped to the table.

**Comment by chrisjlocke** [comment]
While using xplorer2 (file explorer replacement) this has this option. (or one similar)

![image](https://user-images.githubusercontent.com/7872218/112445046-9f9c9b80-8d46-11eb-94f6-f1f6bc3d59bf.png)

So as you flipped tabs, DB4S would use the currently selected tab in the 'database structure' tab to view the table.  I'm not sure how user-intuitive it would be ... but have added the 'enhancement' label so its not lost.
